src/app/core/services/advance-request.service.ts

```
@@ -552,12 +550,17 @@
+      ou_department:
+        advanceRequestPlatform.employee &&
+        advanceRequestPlatform.employee.department &&
+        advanceRequestPlatform.employee.department.name,
+      ou_department_id: advanceRequestPlatform.employee && advanceRequestPlatform.employee.department.id,
       ou_id: advanceRequestPlatform.employee_id,
       ou_org_id: advanceRequestPlatform.org_id,
       ou_sub_department:
-        advanceRequestPlatform.employee.department && advanceRequestPlatform.employee.department.sub_department,
+        advanceRequestPlatform.employee &&
+        advanceRequestPlatform.employee.department &&
+        advanceRequestPlatform.employee.department.sub_department,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)